lib/msf/core/exploit/sqli/postgresqli/common.rb

```diff
@@ -202,6 +202,22 @@ def write_to_file(fname, data)
       raw_run_sql("copy (select '#{data}') to '#{fname}'")
     end
 
+    #
+    # Attempt reading from a file on the filesystem
+    # @param fpath [String] The path of the file to read
+    # @param binary [String] Whether the target file should be considered a binary one (defaults to false)
+    # @return [String] The content of the file if reading was successful
+    #
+    def read_from_file(fpath, binary=false)
+      if binary
+        # pg_read_binary_file returns bytea
+        # an encoder might be needed
```

Comment:
Is this comment still relevant? This isn't something we should leave up in the air before trying to land this in my opinion.